docs/pages/status.js

```diff
@@ -43,7 +43,65 @@ export default function SearchFieldPage({ generatedDocGen }: {| generatedDocGen:
           />
         </MainSection.Subsection>
       </MainSection>
+      <MainSection name="Best practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use Status to communicate a step in a workflow or the state of an item."
+            defaultCode={`
+<Status type="ok" accessibilityLabel="This item is ok" />
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use SVGs or images that resemble the Status’ symbols to denote status."
+            defaultCode={`
+<Icon icon="workflow-status-problem" size="24" accessibilityLabel="This item has an error" />
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Place Status close to the status subject in question to provide context and reference. It can be placed as an inline element or paired side by side as needed.  "
+            defaultCode={`
+<Flex gap={1} alignItems="center">
+  <Status accessibilityLabel="This item is complete" type="ok" />
+  <Text weight="bold" size="300">Campaign complete</Text>
+</Flex>
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Place Status far away from its subject. "
+            defaultCode={`
+<Flex gap={12} direction="column" alignItems="center">
+  <Status accessibilityLabel="This item is paused" type="halted" />
+  <Text weight="bold" size="300">Campaign paused</Text>
+</Flex>
 
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use a title when the status it represents is unique, specific and critical for the user to know."
```

Comment:
props should be denoted with backticks and don't need an article, so
> Use `title` when the status it represents…

Reply:
There was some initial feedback that utilizing prop names may not necessarily be as clear to designers, so we decided to pull is back to something less technical.

Reply:
Done!